config/300-certificate.yaml

```diff
@@ -39,3 +39,45 @@ spec:
   - name: Reason
     type: string
     JSONPath: ".status.conditions[?(@.type==\"Ready\")].reason"
+---
+# As the Certificate controller is using cert-manager `Certificate` informer,
+# the CRD definition of cert-manager `Certificate` is needed in order to
+# successfully start the controller.
```

Comment:
Can we pull this in via the vendor and symlink it here?  see the caching resource.

Reply:
Done